lib/scan.ts

```diff
@@ -136,3 +140,13 @@ export async function scan(options: Options): Promise<PluginResponse> {
     throw new Error(`Could not scan C/C++ project: ${error}`);
   }
 }
+
+function getExcludePatterns(projectRoot: string, dotSnykPath: string) {
```

Comment:
should we create a test for this function?
(To test the integration between parseDotSnyk and parseUnmanagedExclude ?)

Reply:
Yep, I think so.

Reply:
👍